python/paddle/fluid/contrib/slim/tests/save_qat_model.py

```diff
@@ -43,10 +43,11 @@ def parse_args():
         default='',
         help='Saved optimized and quantized INT8 model')
     parser.add_argument(
-        '--quantized_ops',
+        '--ops_to_quantize',
         type=str,
         default='',
-        help='A comma separated list of quantized operators.')
```

Comment:
Not exactly related to the PR itself, but it seems to me, that save_qat_model, qat2_int8_nlp_comparison.py, qat2_int8_image_classification_comparison.py all share similar argument options. Perhaps it would be worth to consider moving shared options to some common `argparse.ArgumentParser()` that would be reused acrossed these 3 files.

Reply:
It is a good idea! We will add it when refactoring the QAT code.

Reply:
Could you not move the parameters in save_qat_model.py? Because it will be copied by users in PaddleSlim, it need to run independently. Thanks